source/docs/http3_upstream.md

```diff
@@ -42,6 +42,12 @@ sent to the mixed pool, and whichever succeeds first will be passed back up to t
 
 Specific Envoy configuration is required in order to enable the previously described components.
 
+#### Alternate Protocols Cache Filter
+
+The Alternate Protocols Cache Filter must be enabled in order for alt-svc headers to be parsed
```

Comment:
These will need reflinks and example configs, but I don't think we want to do that until we finish the functionality and unhide the alt-svc-cache filter - will remind you about it then :-)

Reply:
OK yeah if this is all WIP code that's fine, but once we are ready to ship it we need to move all of this to real docs per @alyssawilk